inputs/org.osbuild.containers

```diff
@@ -163,6 +191,21 @@ class ContainersInput(inputs.InputService):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.mg = MountGuard()
+        self.storage_conf = None
+
+    def map_storage_ref(self, _source, ref, target):
+        if self.storage_conf is None:
```

Comment:
Why not do this in `__init__`?

Reply:
Ah you're right it's not very clear. I can add a comment. But basically we don't want to do this for the sources that don't need it since it may break things